### ISSUE TITLE
"-" character is shown when the Tagline is empty

### ISSUE THREAD

**Issue by iamgabrielma** [WordPress/twentynineteen#451]
If the title is filled out but the tagline is left empty, we can see the "-" character after the title in the front-end:

<img width="873" alt="screen shot 2018-11-01 at 19 26 47" src="https://user-images.githubusercontent.com/3812076/47851747-2d8d2a00-de0c-11e8-8331-523f732a6c54.png">

This seems to appear in `site-description::before`

<img width="931" alt="screen shot 2018-11-01 at 19 29 08" src="https://user-images.githubusercontent.com/3812076/47851848-6f1dd500-de0c-11e8-9a49-4b943e8914a7.png">

**Comment by kjellr** [comment]
Clarifying note here: This bug only happens in the customizer. It was fixed in the front end via #315.

The customizer was likely missed before because the em dash is typically overlapped by the partial edit shortcut icon in that context:  

<img width="772" alt="screen shot 2018-11-01 at 8 42 06 am" src="https://user-images.githubusercontent.com/1202812/47852405-0c97eb00-ddb2-11e8-8f0b-8cafb9e04328.png">

**Comment by iamgabrielma** [comment]
> The customizer was likely missed before because the em dash is typically overlapped by the partial edit shortcut icon in that context

That's right, thanks for clarifying. I happened to see the dash before the customizer icons loaded, then for the screenshot that demonstrates the bug I removed the icons via developer tools.

**Comment by ianbelanger79** [comment]
It appears that adding the "-" (EM Dash) symbol with CSS may not work in this case. The main reason being that it uses `:not(:empty)` to add it before the tagline. This works fine on the front-end, but in the customizer, the site description `<p>` tags are never empty, due to the fact that the "edit" button is added inline inside the site description `<p>` tags.

@allancole and @kjellr what are your thoughts on adding the "-" (EM Dash) symbol inline? I will submit a PR with this approach.

**Comment by kjellr** [comment]
@ianbelanger79 We had it inline originally, but moved it to CSS via https://github.com/WordPress/twentynineteen/pull/129/. If that no longer makes sense, I'm happy to have it inline again.
